applications/SwimmingDEMApplication/python_scripts/swimming_DEM_procedures.py

```diff
@@ -264,7 +264,7 @@ def CreateProblemDirectories(self, main_path, dir_names):
 
     def ControlEcho(self, step, incremental_time, total_steps_expected):
 
-        if incremental_time > self.param.ControlTime:
+        if incremental_time > self.parameters["ControlTime"].GetDouble():
```

Comment:
@GuillermoCasas

Reply:
@GuillermoCasas  was in front of my pc when we did this change